GoogleAnalyticsTracker.Core/TrackerBase.async.cs

```diff
@@ -63,6 +63,12 @@ private void SetRequired(IGeneralParameters parameters)
         {
             parameters.TrackingId = TrackingAccount;
             parameters.ClientId = AnalyticsSession.GenerateSessionId();
+
+            if (string.IsNullOrWhiteSpace(parameters.UserLanguage))
+            {
+                // Note: another way could be CultureInfo.CurrentCulture
```

Comment:
Agree. I am not a real asp.net expert so not sure if the CurrentCulture represents the server culture or takes the culture of the request.